datalad/distributed/create_sibling_ria.py

```diff
@@ -394,7 +400,18 @@ def __call__(url,
         #       - more generally consider store creation a dedicated command or
         #         option
 
-        create_store(SSHRemoteIO(ssh_host) if ssh_host else LocalIO(),
+        io = SSHRemoteIO(ssh_host) if ssh_host else LocalIO()
+        if not io.exists(Path(base_path)) and expect_store:
```

Comment:
For consistency with how the ORA remote asses the existence of a store, a valid `ria-layout-version` file at that path is needed, too. Mere existence of the path itself doesn't make it a useable store. Also: `base_path` could be a file and would pass this test.

So, I think change condition to "`io.read_file(Path(base_path)  / 'ria-layout-version')` doesn't raise anything".